src/AlixBa/Addic7edSubtitles/Helpers/Episode.php

```diff
@@ -64,6 +64,18 @@ public function __construct($episodeFilename)
      *
      * @return string
      */
+    public static function capitalizeShowName($showName)

```

Comment:
I'm fine with your arguments. sanitize is static because it is used in the ShowsUpdater.